docs/_includes/sign-up/sign-up-form.html

```diff
@@ -22,6 +22,11 @@ <h2 id='hosted-signup'>Sign Up to Evaluate Conjur</h2>
         <div class="help-block with-errors"></div>
     </div>
 
+    <div class="form-group required">
+      <div id="recaptcha" class="g-recaptcha" data-sitekey="6Lddji0UAAAAAAiy2M--2e_BNcPop7b_kbJKv5tp"></div>
```

Comment:
Wish we could get this out of code :( Could we pass the site key in through at compilation time and store it in Conjur (or at least the `secrets.yml` file)?

Reply:
Ah yeah I suppose we could work it in at compile time, but not sure if it's worth it since someone who really wanted to get this value only needs to load our site and poke around in the source. There's a separate secret key value for calling the API for token verification. I got the impression that this one was supposed to be semi-public, a bit like the Google Analytics key.

Reply:
I'm fine keeping it in here for now.